advice.c

```diff
@@ -59,7 +87,8 @@ void advise(const char *advice, ...)
 
 	for (cp = buf.buf; *cp; cp = np) {
 		np = strchrnul(cp, '\n');
-		fprintf(stderr,	_("hint: %.*s\n"), (int)(np - cp), cp);
+		fprintf(stderr,	_("%shint: %.*s%s\n"), advise_get_color(ADVICE_COLOR_HINT),
+			(int)(np - cp), cp, advise_get_color(ADVICE_COLOR_RESET));
```

Comment:
I looked and it would appear that this is the first time we are using color on `stderr`. That is not necessarily a bad thing! However, I feel a bit like the wrong person to decide this, as it will affect more than just Git for Windows: it will affect every Git user, also on Linux and macOS.

How would you feel about me taking your current work and submitting it to the Git mailing list to solicit advice?